### ISSUE TITLE
Compress original screenshot images.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-showcase-2022#219]
JetPack is converting screenshots to WebP, greatly reducing image size but the original pngs should also be compressed. Can we run a script to compress the screenshots?

Take for example: 

<img width="300" alt="Screenshot 2023-10-04 at 12 15 53 PM" src="https://github.com/WordPress/wporg-showcase-2022/assets/1657336/e130d43b-8b64-403d-9058-37b3b04e3810">

Site Link :https://wordpress.org/showcase-v2/opus-one/

**Current**
Media library `.png` is `4mb`.
Outputted `.webp` is `1.2mb`

I compressed that image and was able to cut size by ~60% with negligible visible difference.

Media library `.png` is `1.8mb`.
Outputted `.webp` is `482kb`.

Page load improved by ~400ms (on my connection) by making an update to this one image. On single pages, there are 4 of these images.

**Comment by StevenDufresne** [comment]
This also relates to: https://github.com/WordPress/wporg-showcase-2022/pull/175#issuecomment-1746062565

**Comment by ryelle** [comment]
When I updated the "how to screenshot" wiki I added a note about using something like [ImageOptim](https://imageoptim.com/mac). Ideally this would be done on upload — but now we have many uploaded images already, so a script to update them would be nice, especially with that much improvement.

The script would also need to update each site though, because once CDN'd, the image would not be re-synced.

> No cache invalidations. Currently, static assets are tied to the public version of WordPress, Jetpack, or WooCommerce that you’re using. For images, **if you want to “refresh” an image, you will need to change its file name**. Adding random query arguments, commonly known as “cache busters,” will not work.
> —[source](https://jetpack.com/support/site-accelerator/)

So for every site: get the desktop & mobile image, optimize it, reupload it to the media library, and update the screenshot meta fields with the new attachment ID.

Not impossible, but could be tricky.
